### PR TITLE
Handle undefined response from RDT backend (FE-607)

### PR DESCRIPTION
**Pull request by hbenl**

### CODE REVIEW

File: src/ui/components/SecondaryToolbox/ReactDevTools.tsx [hunk 1]
```diff
@@ -172,7 +172,9 @@ class ReplayWall implements Wall {
 
     if (response.returned) {
       const result: any = await response.returned.getJSON();
-      this._listener?.({ event: result.event, payload: result.data });
+      if (result) {
+        this._listener?.({ event: result.event, payload: result.data });
+      }
       return result;
```
Comment (bvaughn) [inline src/ui/components/SecondaryToolbox/ReactDevTools.tsx:178]: The whole `sendRequest` method looks weird to me. It returns _either_ a `result` (which might be undefined) _or_ `response.returned` (which also might be undefined).

I'm not sure what it's supposed to be doing? I think maybe @jasonLaster would be better suited to review this as the author of the code originally (#3861)?

Comment (hbenl) [reply src/ui/components/SecondaryToolbox/ReactDevTools.tsx:178]: That `return response.returned` doesn't make sense (it's only executed if `response.returned` is falsy) so I just removed it.

Comment (hbenl) [reply src/ui/components/SecondaryToolbox/ReactDevTools.tsx:178]: And I don't think @jasonLaster has more insights on this, so I'm just gonna land it.

Comment (bvaughn) [reply src/ui/components/SecondaryToolbox/ReactDevTools.tsx:178]: I dunno. Seems like a sketchy function that's exactly the sort of thing we should invest in reviewing.

Jason wrote the function originally. Why wouldn't he have insight?

Comment (jasonLaster) [reply src/ui/components/SecondaryToolbox/ReactDevTools.tsx:178]: going to go on record and say i do not have additional insights... sorry to disappoint

**Pull request merged**